src/Form/FormAnalyzer.js

```diff
@@ -75,24 +105,25 @@ class FormAnalyzer {
         shouldCheckUnifiedForm = false,
         shouldBeConservative = false
     }) {
-        const matchesNegative = string === 'current-password' || negativeRegex.test(string)
+        const matchesLogin = string === 'current-password' || loginRegex.test(string)
 
         // Check explicitly for unified login/signup forms. They should always be negative, so we increase signal
-        if (shouldCheckUnifiedForm && matchesNegative && strictPositiveRegex.test(string)) {
-            this.decreaseSignalBy(strength + 2, `Unified detected ${signalType}`)
+        if (shouldCheckUnifiedForm && matchesLogin && strictSignupRegex.test(string)) {
+            this.signals.push(`hybrid form: ${signalType}`)
+            this.isHybrid = true
```

Comment:
If we detect that it's a hybrid form, short circuit things. Actually, I'm wondering if we should also stop scanning the form at this stage 🤔. Feedback welcome, but as things stand now once this is set we never un-set it so the rest of the form scanning doesn't have any effect.